src/node_quic_socket.h

```diff
@@ -172,9 +172,9 @@ class QuicSocket : public AsyncWrap,
       QuicCID* scid,
       const sockaddr* addr);
 
-  void IncrementSocketAddressCounter(const sockaddr* addr);
-  void DecrementSocketAddressCounter(const sockaddr* addr);
-  size_t GetCurrentSocketAddressCounter(const sockaddr* addr);
+  void IncrementSocketAddressCounter(const sockaddr_storage* addr);
+  void DecrementSocketAddressCounter(const sockaddr_storage* addr);
+  size_t GetCurrentSocketAddressCounter(const sockaddr_storage* addr);
```

Comment:
I think all the APIs should keep taking `const sockaddr*` or `const sockaddr&`, because they should still work when a `sockaddr_in*` or `sockaddr_in6` is passed in (which are not as large as `sockaddr_storage`)

Reply:
I took `const sockaddr_storage*` here to avoid creating `sockaddr_storage` from `sockaddr` as we can't `reinterpret_cast` them directly. Not sure if there are better solutions.